libsolidity/formal/SymbolicState.cpp

```diff
@@ -142,13 +142,26 @@ smtutil::Expression SymbolicState::txMember(string const& _member) const
 	return m_tx.member(_member);
 }
 
+// TODO: Remove this function in 0.9.0
+smtutil::Expression SymbolicState::evmParisConstraints() const
```

Comment:
Are we actually going to remove the whole function? The lower bound on prevrandao looks like something we'd like to keep.

Reply:
[](url)

Reply:
Indeed. And actually, `txTypeConstraints` is not the best place to call it either since the evm restriction is not a type constraint, but I couldn't find a better place. So this should be refactored in `0.9.0` anyway.

Reply:
The comment is wrong then. It needs to be reworded.

Reply:
I reworded it. Please see if it is better now :)

Reply:
Looks fine now.